lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -554,7 +551,17 @@ def _partition_pdf_batched_named(
             with LogTime("pdfminer_supplement"):
                 for d, p in zip(deformable_layout, pdfminer_layout):
                     self._supplement_text(d, p)
-
+        # TODO: optimize this to make pdfminer also streamed so we can process each page in sequence without
+        # having to double-convert the document
```

Comment:
Just to make sure I understand correctly, this will double-convert even in cases when you aren't using pdfminer, right? That may be fine, but we need to do some performance measurements, I think. Can we get this on one of the test instances and run the load tests + measure latency?

Reply:
Will work on this after merging this in